dm/syncer/dml.go

```diff
@@ -84,6 +86,8 @@ func (s *Syncer) genAndFilterInsertDMLs(tctx *tcontext.Context, param *genDMLPar
 		originalDataSeq = param.originalData
 		columns         = param.columns
 		ti              = param.sourceTableInfo
+		extendData      = param.extendData
+		extendValues    = param.extendValues
```

Comment:
seems we can remove `extendValues`, the usage can be changed to 

if extendData != nil {originalDataSeq = extendData}